### ISSUE TITLE
Facebook Page Widget not displaying

### ISSUE THREAD

**Issue by DIYselflearner** [Automattic/jetpack#10544]
Having such a hard time with the Facebook Page widget both on latest version Firefox and Chrome. Ticket I opened with Automattic is long. Basically this widget is affected in all kinds of ways by any kind of built-in browser tracking and/or browser customizations such as ad blockers. After much back and forth and trail and error, I discovered that with Firefox it's the internal tracking blocker that causes widget to not display. On Chrome, no matter what I did I was unable to duplicate. Still won't show on Chrome. First time at this so don't even know I'm submitting correctly. Utterly frustrated.

**Comment by jeherve** [comment]
I can see that you've been in touch with our support team already (internal references 1535600-zen and 1496936-zen).

I see they had some suggestions, and I am glad you could at least figure out what was causing the issue in Firefox:

> I discovered that with Firefox it's the internal tracking blocker that causes widget to not display. 

That's indeed to be expected. Firefox's built-in tracking tools are built to block all third-party trackers, and that includes Facebook. You can learn more about this here:

- https://www.mozilla.org/en-US/firefox/63.0.1/tracking-protection/start/?step=2&newtab=true&variation=2
- https://support.mozilla.org/en-US/kb/tracking-protection?redirectlocale=en-US&redirectslug=tracking-protection-pbm

Your site's visitors using Firefox have the option to allow tracking on your site:

<img width="565" alt="screenshot 2018-11-06 at 12 54 52" src="https://user-images.githubusercontent.com/426388/48062977-f0d28180-e1c3-11e8-8e0e-2d5cc05eb451.png">

> Basically this widget is affected in all kinds of ways by any kind of built-in browser tracking and/or browser customizations such as ad blockers.

Note that this isn't specific to the widget included in Jetpack. If you were to add the Facebook Page Plugin on your own by copying and pasting code from Facebook, without using any plugin, you would still experience the same result. This is done on purpose by Firefox, as they aim to stop third-parties from tracking your activity on different websites; Facebook's Page Plugin tracks your activity, so it is blocked.

Google Chrome, however, does not include such functionality yet. That is why the page plugin works well on your site when viewed in Chrome. Here is how it looks on desktop:

<img width="1227" alt="screenshot 2018-11-06 at 13 04 05" src="https://user-images.githubusercontent.com/426388/48063155-76eec800-e1c4-11e8-9970-6010d96a8de3.png">

And on Chrome for Android:

![screenshot_20181106-130535](https://user-images.githubusercontent.com/426388/48063253-bf0dea80-e1c4-11e8-811f-35aad1e64507.png)

Since you don't see the Page Plugin on your own installation of Chrome, I suspect something else is interferring on your computer. As you mentioned and as my colleagues had suggested, privacy and ad-blocker extensions can block Facebook resources on your site, to protect your privacy. You've tried to deactivate the extensions you use in Chrome (Disconnect and Adblock plus), but that did not help. If you do not use any other extension in your browser and still experience issues when those 2 extensions are deactivated, that leaves us with 2 options:

1. Something else on your computer blocks access to Facebook's tracking scripts. I would lean towards an anti-virus, since more and more anti-virus software includes anti-tracking options. That said, if you can see the Facebook Page plugin in Firefox when deactivating the privacy features, that means that no anti-virus is blocking resources here, the problem is limited to Chrome.
2. Something else in Chrome blocks access to Facebook's tracking scripts. If that is the case, I would suggest that you review your list of extensions to make sure that no other extension blocks access to those tracking scripts.

Hopefully this helps you get to the bottom of this.

**Comment by DIYselflearner** [comment]
I replied via email on other points but since I was forced to create an account here and report something I've already detailed... This particular answer is a non-answer. You're just repeating what I tested over and over on my OWN TIME, because Jetpack provided zero guidance. I went through this with your social icons as well. Got them to show both in Chrome and Firefox. Those have the same tracking/privacy concerns, correct? Displays fine now on my Chrome. So how come FB widget doesn't?
